pkg/network/netlink/conntracker.go

```diff
@@ -153,8 +179,11 @@ func newConntrackerOnce(cfg *config.Config) (Conntracker, error) {
 func (ctr *realConntracker) GetTranslationForConn(c network.ConnectionStats) *network.IPTranslation {
 	then := time.Now().UnixNano()
 	defer func() {
-		ctr.stats.gets.Inc()
+		conntrackerTelemetry.getsTotal.Inc()
 		ctr.stats.getTimeTotal.Add(time.Now().UnixNano() - then)
+		if gets := conntrackerTelemetry.getsTotal.Load(); gets > 0 {
+			conntrackerTelemetry.nanoSecondsPerGet.Set(float64(ctr.stats.getTimeTotal.Load() / gets))
+		}
```

Comment:
Should we be using a histogram for this, rather than computing these values ourselves? https://prometheus.io/docs/practices/histograms/

Reply:
added a few histograms where possible